### ISSUE TITLE
Awkward Scrolling

### ISSUE THREAD

**Issue by NoahSimonCDI** [ITPNYU/mc-reservation-tool#243]
When a space and day is selected, there is something confusing going on with scrolling. There ends up being two scroll bars, one for the embedded google calendar, the other for the page itself. If scrolled all the way to the end of the day on the calendar section, the main portion of the page seems to not be able to scroll. So you have to scroll the calendar back up towards 1am, and then you can scroll the page which allows you to continue the process. This is more confusing in Safari, where (may be my settings) the scroll bars only appear when scrolling. In Chrome, this is easier to see and hence navigate. But definitely look at this in different browsers, as it may look and behave differently in each.

**Comment by ecb344** [comment]
I agree with Noah here, it is not easy to navigate the booking calendar. In addition to the scrolling difficulty, it is also hard to visualize a block of time because the embedded section is quite small compared to the calendar its self (vertically). Is it possible to make the embedded section larger (vertically) so that one can scroll the entire page down (seeing less of the banner) to see more of the calendar?

![image](https://github.com/ITPNYU/mc-reservation-tool/assets/173960321/8ba92bec-520b-4c55-b480-f3f79c411614)
